Agents/FenlandTrajectoryAgent/README.md

```diff
@@ -95,15 +95,7 @@ The files for example http request are displayed at [SendHTTP] folder.
 - Post request to instantiate all GPS trajectory data to KG
 > ./gpstasks/fenlandtrajectoryagent/process_and_instantiate
 
-Additionally, services above can be triggered using [CURL commands] from a bash terminal. This method provides a straightforward and scriptable way to interact with the agent. An example CURL command used to load the GPS trajectory files is displayed as follows. In this example, the stack is assumed at port 3840
-
-```
-curl -X POST http://localhost:3840/fenland-trajectory-agent/gpstasks/fenlandtrajectoryagent/load_and_preprocess \
-     -H "Content-Type: application/json" \
-     -d '{"file_path":"/app/agent/raw_data/gps_target_folder"}'
-
-```
-
+Additionally, services above can be triggered using Client URL (CURL) from a bash terminal. This method provides a straightforward and scriptable way to interact with the agent. An example CURL command used to load the GPS trajectory files is displayed in [CURL commands folder]. In this example, the stack is assumed at port 3840.
```

Comment:
Please remove the sentence "In this example, the stack is assumed at port 3840.". It is redundant and bound to become out of sync.

Reply:
Yes! This was addressed at commit [c1f26e82d20cd41106365111f6517932562651ba]